conda_build/build.py

```diff
@@ -1039,7 +1043,7 @@ def check_external():
 
 
 def build_tree(recipe_list, config, build_only=False, post=False, notest=False,
-               need_source_download=True, need_reparse_in_env=False):
+               need_source_download=True, need_reparse_in_env=False, verify=True):
```

Comment:
same as above: verify should probably be a config parameter, rather than a new argument here. Take a look at how test is done for reference.